spec/weak-ref.html

```diff
@@ -89,6 +89,18 @@ <h1>Properties of the WeakRef Prototype Object</h1>
       <li>does not have a [[Target]] internal slot.</li>
     </ul>
 
+    <emu-clause id="sec-weak-ref.prototype.constructor" normative-optional>
+      <h1>WeakRef.prototype.constructor</h1>
+
+      <p>The initial value of `WeakRef.prototype.constructor` is the intrinsic object %WeakRef%.</p>
+
+      <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.</p>
+
+      <emu-note type="editor">
+        This section is to be treated identically to the "Annex B" of ECMA-262, but to be written in-line with the main specification.
```

Comment:
can you elaborate on this conclusion?

Reply:
The idea here is to put Annex B inline to make it easier to read. We do this in ECMA-402. We discussed this earlier today in the TC39 meeting; delegates can see the draft notes. Do you have any concerns with this idea?

Reply:
It seems fine, but that's a pretty massive editorial change discussed with no editors present, and wasn't what I expected from a slideless presentation called "reform".

Reviewing the notes gives me a bit more clarity, but I'd still hope to see a few examples of inline normative optional land in the main spec before a proposal added one (that shouldn't block this PR ofc)

Reply:
Yeah, the plan is that I'll file an issue in ecma262 describing the plan, and we'll work from there in PRs. This PR can be thought of as a really early first draft, editorially; we just need to get it off the ground here and have the observable semantics specified.